src/components/context/UserContextProvider.tsx

```
@@ -0,0 +1,45 @@
+// @ts-ignore

```

What's being ignored? I'd rather we didn't ignore errors and if we do then provide a comment as to why